docs/using-with-rails.md

```diff
@@ -38,3 +38,20 @@ Or to add content to `<head>`, for stylesheets or similar:
 ```
 
 Check out the [full list of blocks](template-blocks.md) you can use to customise the template.
+
+## SRI 
```

Comment:
Do the browser bugs we encountered need mentioning? And the gem you wrote to fix them?

Reply:
Good point, I'll update the readme.

Reply:
done in a rebase.